wfe2/wfe.go

```diff
@@ -1548,7 +1549,22 @@ func (wfe *WebFrontEndImpl) Certificate(ctx context.Context, logEvent *web.Reque
 		requesterAccount = acct
 	}
 
+	requestedChain := 0
 	serial := request.URL.Path
+
+	// An alternate chain may be requested with the request path {serial}/{chain}, where chain
+	// is a single digit - an index into wfe.certificateChains[aiaIssuerURL]. If a specific chain is
+	// not requested, then it defaults to zero - the default certificate chain for that issuer.
+	if len(serial) > 3 && serial[len(serial)-2] == '/' {
```

Comment:
Instead of indexing on the string, could you use `strings.SplitN` here? There's a similar example at https://github.com/letsencrypt/boulder/blob/e92c5c66ea89517c8d3422b09c3a355c53430a40/wfe2/wfe.go#L1982-L1997.

Reply:
Good point, I felt uneasy doing this kind of slicing, but an alternative didn't come into my head.